From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: ZeoAlt <zeonightyt@gmail.com>
Date: Tue, 28 Dec 2021 16:56:12 +0000
Subject: [PATCH] Debug Packets


diff --git a/src/main/java/net/minecraft/network/Connection.java b/src/main/java/net/minecraft/network/Connection.java
index 81dde0efc1a06420c0791520b9e40b24dd1f0318..270a882c58fd1a17c4c2afa76afd1ca1826eb31a 100755
--- a/src/main/java/net/minecraft/network/Connection.java
+++ b/src/main/java/net/minecraft/network/Connection.java
@@ -331,6 +331,7 @@ public class Connection extends SimpleChannelInboundHandler<Packet<?>> {
     // Paper end
 
     public void send(Packet<?> packet) {
+        LOGGER.info(packet)
         this.send(packet, (GenericFutureListener) null);
     }
 
